apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -314,8 +314,8 @@ const TicketContent = ({ ticket }) => {
                                         <Typography.Link style={TICKET_CARD_LINK_STYLE}>
                                             <TicketUserInfoField
                                                 user={{
-                                                    name: get(ticket, 'clientName'),
-                                                    phone: get(ticket, 'clientPhone'),
+                                                    name: get(ticket, ['contact', 'name']),
```

Comment:
How about:

```
const name = get(ticket, 'clientName')
.....
name: get(ticket, ['contact', 'name'], name)
```

Reply:
`clientName` and `clientPhone` are taken if there is no `contactId` and are conditionally rendered below in the code